content/blog/deploy-stateful-mongodb-application-on-kubernetes-clusters-in-hpe-greenlake-for-containers-1.md

```diff
@@ -0,0 +1,547 @@
+      "health" : 1,
+      "state" : 1,
+      "stateStr" : "PRIMARY",
+      "uptime" : 459,
+      "optime" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDate" : ISODate("2022-07-20T09:26:13Z"),
+      "syncingTo" : "",
+      "syncSourceHost" : "",
+      "syncSourceId" : -1,
+      "infoMessage" : "",
+      "electionTime" : Timestamp(1658308953, 2),
+      "electionDate" : ISODate("2022-07-20T09:22:33Z"),
+      "configVersion" : 4,
+      "self" : true,
+      "lastHeartbeatMessage" : ""
+    },
+    {
+      "_id" : 1,
+      "name" : "pce-mongodb-replica-1.pce-mongodb:27017",
+      "health" : 1,
+      "state" : 2,
+      "stateStr" : "SECONDARY",
+      "uptime" : 61,
+      "optime" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDurable" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDate" : ISODate("2022-07-20T09:26:13Z"),
+      "optimeDurableDate" : ISODate("2022-07-20T09:26:13Z"),
+      "lastHeartbeat" : ISODate("2022-07-20T09:26:18.007Z"),
+      "lastHeartbeatRecv" : ISODate("2022-07-20T09:26:18.010Z"),
+      "pingMs" : NumberLong(0),
+      "lastHeartbeatMessage" : "",
+      "syncingTo" : "pce-mongodb-replica-0.pce-mongodb:27017",
+      "syncSourceHost" : "pce-mongodb-replica-0.pce-mongodb:27017",
+      "syncSourceId" : 0,
+      "infoMessage" : "",
+      "configVersion" : 4
+    },
+    {
+      "_id" : 2,
+      "name" : "pce-mongodb-replica-2.pce-mongodb:27017",
+      "health" : 1,
+      "state" : 2,
+      "stateStr" : "SECONDARY",
+      "uptime" : 31,
+      "optime" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDurable" : {
+        "ts" : Timestamp(1658309173, 1),
+        "t" : NumberLong(1)
+      },
+      "optimeDate" : ISODate("2022-07-20T09:26:13Z"),
+      "optimeDurableDate" : ISODate("2022-07-20T09:26:13Z"),
+      "lastHeartbeat" : ISODate("2022-07-20T09:26:18.007Z"),
+      "lastHeartbeatRecv" : ISODate("2022-07-20T09:26:18.536Z"),
+      "pingMs" : NumberLong(0),
+      "lastHeartbeatMessage" : "",
+      "syncingTo" : "pce-mongodb-replica-1.pce-mongodb:27017",
+      "syncSourceHost" : "pce-mongodb-replica-1.pce-mongodb:27017",
+      "syncSourceId" : 1,
+      "infoMessage" : "",
+      "configVersion" : 4
+    }
+  ],
+  "ok" : 1,
+  "$clusterTime" : {
+    "clusterTime" : Timestamp(1658309173, 1),
+    "signature" : {
+      "hash" : BinData(0,"AAAAAAAAAAAAAAAAAAAAAAAAAAA="),
+      "keyId" : NumberLong(0)
+    }
+  },
+  "operationTime" : Timestamp(1658309173, 1)
+}
+rs0:PRIMARY>
+```
+
+The **members** section of the status output shows three replicas. The pod *pce-mongodb-replica-0* is listed as the **PRIMARY** replica, while the other two pods, *pce-mongodb-replica-1* & *pce-mongodb-replica-2*, are listed as the **SECONDARY** replicas.
```

Comment:
Should PRIMARY and SECONDARY be all caps? Is that how they are actually named or shown on a screen? Try to keep consistent throughout the blog post.

Reply:
Good catch. I will change them to Primary and Secondary to be more consistent.